canonical-websites/tutorials.ubuntu.com
# Adds a gci badge

This adds a way to promote Google Code-in tutorials and publish them even though they could be improved. This PR:

- adds an icon to tutorials cards with the `gci` tag
- adds a note at the top of `gci` tagged tutorials
- puts 3 GCI tutorials on the homepage (by removing the `hidden` tag)

## QA

* Go to http://tutorials.ubuntu.com-pr-577.run.demo.haus/
* Ensure the GCI icon is present on 3 cards
* Click on one of them
* Ensure the first element after the title of the first step is a blue GCI-related admonition 

## Screenshots

![screenshot from 2017-12-20 17-43-43](https://user-images.githubusercontent.com/18480003/34218047-6da43d98-e5ad-11e7-8a3d-8c661f9cbfb9.png)

![screenshot from 2017-12-20 17-45-00](https://user-images.githubusercontent.com/18480003/34218108-92c83d5e-e5ad-11e7-8b6d-d4dd1ab976e1.png)


## Future

Ideally, this mechanism will be made more flexible and give us a new way of surfacing special content.
